src/styles/index.ts

```diff
@@ -2796,13 +2796,29 @@ const styles = (theme: ThemeColors) =>
             fontWeight: FontUtils.fontWeight.bold,
         },
 
+        borderedContentCard: {
+            borderWidth: 1,
+            borderColor: theme.border,
+            borderRadius: variables.componentBorderRadiusNormal,
+        },
+
+        borderedContentCardFocused: {
```

Comment:
In what cases is the card focused? It's just there to display information, so I don't quite understand the focus behavior.

Reply:
![Screenshot 2024-06-04 at 15 18 01](https://github.com/Expensify/App/assets/44364829/9d2f5903-9f73-4225-8223-3e8a5a65e60b)

In this case, when it wraps the active subscription type

Reply:
Great idea, I removed it and used the existing `borderColorFocus` style instead, let me know what you think!

Reply:
Love reusing existing styles! Thanks!

Reply:
Very helpful advice from your side once again, thanks!